samples/Audio/SimpleAudio/SimpleAudio.Windows/SimpleAudio.Windows.csproj

```diff
@@ -1,58 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net8.0-windows7.0</TargetFramework>
+    <TargetFramework>net8.0-windows</TargetFramework>
     <ApplicationIcon>Resources\GameIcon.ico</ApplicationIcon>
     <OutputType>WinExe</OutputType>
     <RootNamespace>SimpleAudio</RootNamespace>
     <OutputPath>..\Bin\Windows\$(Configuration)\</OutputPath>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
-  </PropertyGroup>
-  <PropertyGroup>
     <StrideCurrentPackagePath>$(MSBuildThisFileDirectory)..\SimpleAudio.sdpkg</StrideCurrentPackagePath>
     <StrideProjectType>Executable</StrideProjectType>
     <StridePlatform>Windows</StridePlatform>
-    <StrideBuildProfile>Windows</StrideBuildProfile>
```

Comment:
What was the use of this property before? Could it have been used by TeamCity when running the samples as tests?

Reply:
Yes it was for AssetCompiler as `profile` argument value, but this option was removed

Reply:
Commit : https://github.com/stride3d/stride/commit/9f8f407d4d5796fc3dbf562849ec4f8ea37c1e47